config/locales/views/layouts/_header/en.yml

```diff
@@ -14,6 +14,7 @@ en:
           tools: "Tools"
           places: "Places"
           kits: "Kits"
+          users_map: "Users map"
```

Comment:
Again here, I think we'd like input from @ebarry on naming this something other than "Users map".